pkg/cli/clients/clients.go

```diff
@@ -60,6 +60,12 @@ type DeploymentOptions struct {
 	ProgressChan chan<- ResourceProgress
 }
 
+type DeleteOptions struct {
```

Comment:
The `DeleteOptions` struct is missing documentation comments. Consider adding godoc comments for the struct and its fields.